UI/window-basic-main.cpp

```diff
@@ -2138,6 +2138,82 @@ void OBSBasic::OBSInit()
 
 	/* ----------------------------- */
 	/* add custom browser docks      */
+#if BROWSER_AVAILABLE && YOUTUBE_ENABLED
+#define YOUTUBE_VIDEO_URL "://studio.youtube.com/video/"
```

Comment:
Use `static constexpr` for string constants only used in the current compilation unit.

Reply:
Done.

Reply:
This is now resolved.